src/SmartLock.cpp

```diff
@@ -30,32 +30,40 @@ std::ostream& operator<< (std::ostream& lhs, const SmartLock::lock_t& rhs) {
   return lhs;
 }
 
-#if defined(HAS_WIRING_PI)
+#if defined(HAS_PIGPIO)
 
-#include <wiringPi.h>
+#include <pigpio.h>
 
-const unsigned UNLOCKED_LIGHT = 14;
-const unsigned LOCKED_LIGHT = 13;
+const unsigned UNLOCKED_LIGHT = 11;
+const unsigned LOCKED_LIGHT = 10;
 
 void pi_unlock() {
-  digitalWrite(LOCKED_LIGHT, LOW);
-  digitalWrite(UNLOCKED_LIGHT, HIGH);
+  gpioWrite(LOCKED_LIGHT, PI_LOW);
+  gpioWrite(UNLOCKED_LIGHT, PI_HIGH);
 }
 
 void pi_lock() {
-  digitalWrite(UNLOCKED_LIGHT, LOW);
-  digitalWrite(LOCKED_LIGHT, HIGH);
+  gpioWrite(UNLOCKED_LIGHT, PI_LOW);
+  gpioWrite(LOCKED_LIGHT, PI_HIGH);
 }
 
 void pi_clear() {
-  digitalWrite(UNLOCKED_LIGHT, LOW);
-  digitalWrite(LOCKED_LIGHT, LOW);
+  gpioWrite(UNLOCKED_LIGHT, PI_LOW);
+  gpioWrite(LOCKED_LIGHT, PI_LOW);
+}
+
+void pi_shutdown() {
+  gpioTerminate();
 }
 
 void pi_init() {
-  wiringPiSetup();
-  pinMode(LOCKED_LIGHT, OUTPUT);
-  pinMode(UNLOCKED_LIGHT, OUTPUT);
+  if (gpioInitialise() < 0) {
+    ACE_ERROR((LM_ERROR, "ERROR: pigpio initialisation failed\n"));
```

Comment:
```suggestion
    ACE_ERROR((LM_ERROR, "ERROR: pigpio initialization failed\n"));
```
Even if function names are en-GB, error message should be en-US for consistency (or log the function name as-is)